libs/component-sample/src/lib/feature/side-navigation-sample/side-navigation-sample.component.ts

```diff
@@ -289,19 +299,28 @@ export class SideNavigationSampleComponent implements AfterViewInit, OnInit {
     this.change.detectChanges();
 
     this.activeRoute.queryParams.subscribe(queryParams => {
+      console.log(queryParams)
       if (queryParams.item) {
         this.pageHeader = queryParams.item;
         if (this.model.navigationLinks) {
           this.findItemByQueryString(this.model.navigationLinks);
           this.sideNav.select(this.selectedId);
+          this.filtersAccordion.toggle();
         }
       } else {
-        this.pageHeader = 'Unknown';
+        this.pageHeader = 'Select by Domain';
+      }
+      console.log(this.navigationAccordion.expanded)
+      if(!this.filtersAccordion.expanded) {
```

Comment:
Refactor for speed and readability by removing from ngAfterViewInit so it isn't called after any change

Reply:
I've refactored the code here, but I didn't remove it from ngAfterViewInit. I found a different solutions. Let me know if the alternative is okay.